### PR TITLE
This PR adds Arbitration structure to the project

### PR DESCRIPTION
**Pull request by abhinav-anshul**
Adds : 
1. Arbitrations Header, icons, buttons, and overall structure.
2. Minor CSS fixes.
3. Dummy links to the side panel(for dev purposes only).

### CODE REVIEW

File: src/components/Dashboard/DashboardPage.js [hunk 1]
```diff
@@ -124,30 +129,145 @@ function DashboardPage() {
...
+                      gridTemplateColumns: '1fr 3fr',
+
+                      gridColumnGap: '1rem',
+                      padding: '1rem 0rem',
                     }}
-                    wide
                   >
-                    DASHBOARD
-                  </Button>
+                    <div
```
Comment (koshikraj) [inline src/components/Dashboard/DashboardPage.js:155]: This left-side Panel divs are different in Arbitrations and Dashboard page. This should be uniform throughout the project and reusable. The panel should not be called from every page but only maintain in any root file similar to the earlier `MenuPanel.js` which was called only in `MenuView.js`.

Comment (abhinav-anshul) [reply src/components/Dashboard/DashboardPage.js:155]: 1. I am aware of the issue, I am creating one parent component for the side panel as well as the header architecture, which would render all the children (dashboard/profile/arbitrations).

2. The difference in the panel is due to the fact we are moving away from the button based panel(arbitrations) to div(dashboard) based panel, I will address this in the next push.

File: src/NavPanel.js [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+import DashboardIcon from './assets/DashboardIcon.svg'
+import ProfileIcon from './assets/ProfileIcon.svg'
+import SettingsIcon from './assets/SettingsIcon.svg'
+import DashboardMain from './components/Dashboard/DashboardMain'
+import ArbitrationsPage from './components/Arbitrations/ArbitrationsPage'
+
+import '../src/reset.css'
+
+function NavPanel() {
+  const [navSettings, setNavSettings] = useState('unset')
```
Comment (koshikraj) [inline src/NavPanel.js:15]: Use `bool` value (true, false) instead of `set`

**Comment by koshikraj** [comment]
@abhinav-anshul the nav panel navigation isn't functioning after the recent commit.

**Comment by koshikraj** [comment]
@abhinav-anshul the routing doesn't work properly. The URL route doesn't change on click and the arbitration page is all messed up. Look into it. I can't merge this before that is fixed.

**Comment by koshikraj** [comment]
@abhinav-anshul resolve the merge conflicts, I will merge this PR.

**Comment by koshikraj** [comment]
LGTM

**Review by koshikraj — APPROVED** [review]
LGTM

**Pull request merged**